TYPO3.Flow/Classes/TYPO3/Flow/Object/DependencyInjection/ProxyClassBuilder.php

```diff
@@ -11,13 +11,16 @@
  * source code.
  */
 
+use Doctrine\ORM\Mapping as ORM;
+use TYPO3\Flow\Annotations as Flow;
+use TYPO3\Flow\Configuration\ConfigurationManager;
 use TYPO3\Flow\Object\Configuration\Configuration;
 use TYPO3\Flow\Object\Configuration\ConfigurationArgument;
 use TYPO3\Flow\Object\Configuration\ConfigurationProperty;
+use TYPO3\Flow\Object\Exception as ObjectException;
+use TYPO3\Flow\Object\Proxy\ProxyClass;
+use TYPO3\Flow\Reflection\MethodReflection;
 use TYPO3\Flow\Utility\Arrays;
-use TYPO3\Flow\Configuration\ConfigurationManager;
-use Doctrine\ORM\Mapping as ORM;
-use TYPO3\Flow\Annotations as Flow;
```

Comment:
result of a `alt + O` (Optimize Imports) call in PhpStorm. Lines 20, 21 & 22 are new